parrot/src/pfs_main.cc

```diff
@@ -282,6 +288,8 @@ static void show_help( const char *cmd )
 	printf( " %-30s Disable the given service.\n", "--disable-service");
 	printf( " %-30s Make flock a no-op.\n", "--no-flock");
 	printf("\n");
+	printf("Filesystem Options:\n");
+	printf( " %-30s Mount an ext[234] disk image read-only at /ext_n.\n", "--ext-image=<image>");
```

Comment:
I believe this option has also changed. I am mounting it using `--ext /tmp/test.img=/cvmfs`